manifests/rules/samba.pp

```diff
@@ -0,0 +1,17 @@
+# manage in samba/ctdb
+class nftables::rules::samba {
+  nftables::rule {
+    'default_in-netbios_tcp':
+      content => 'tcp dport {139,445} accept',
+  }
+
+  nftables::rule {
+    'default_in-netbios_udp':
+      content => 'udp dport {137,138} accept',
+  }
+
+  nftables::rule {
+    'default_in-ctdb':
```

Comment:
If CTDB is not strictly necessary to run a Samba server I'd create a separate rule class for it (`nftables::rules::ctdb`) and move the responsibility of including it or not to the user of the module.

Reply:
I was rather thinking to have this parameterized with a `boolean` (working on that, that's why I switched the PR to draft).

IMHO A ctdb rule alone makes little sense as you can't run it without samba.